### ISSUE TITLE
Crash when creating Vulkan Graphics Pipeline

### ISSUE THREAD

**Issue by Andreyogld3d** [GPUOpen-Tools/RGA#49]
Hi! RGA crashes when i try to compiling Vulkan Graphics Pipeline.

I attached RGA project, but may be need to change path to resources
Archive contains the soure code of HLSL shaders.

[Particles.zip](https://github.com/GPUOpen-Tools/RGA/files/3522547/Particles.zip)

BR, Andrey

**Comment by AmitBM** [comment]
Hi Andreyogld3d,

Thanks for reporting this issue. Can you please attach here the source files that you used and the .gpso file? The Dropbox link does not seem to work for us.

Please note that RGA does not yet support HLSL for Vulkan (only GLSL, SPIR-V).

**Comment by Andreyogld3d** [comment]
Hi @AmitBM , i have attached files.

> Please note that RGA does not yet support HLSL for Vulkan (only GLSL, SPIR-V).
I know, inside archive I have attached spir-v files and original HLSL sources with *.cmd files for compilation.  If you need to fix some errors and do experemenets.

**Comment by AmitBM** [comment]
Thanks, Andreyogld3d.

We will aim to provide a fix for this in the next release.

**Comment by AmitBM** [comment]
Hi Andreyogld3d,

The crash should be fixed with RGA 2.3 that was released today. Please note that support hasn't yet been added for HLSL in Vulkan mode.

**Comment by Andreyogld3d** [comment]
Hi @AmitBM , I have message in Build Output:
```
-----------------------------------------------
Building Vulkan project "Particles" for gfx1010
-----------------------------------------------
rga.exe -s vulkan --isa "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\Output\Clone0\disassem.txt" --parse-isa --line-numbers --analysis "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\Output\Clone0\resourceUsage.csv" -b "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\Output\Clone0\codeobj.bin" --log "C:\Users\AndreyOGL_D3D\AppData\Roaming\RadeonGPUAnalyzer\rga-cli-20191015-002351.log" --session-metadata "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\Output\Clone0\gfx1010_cliInvocation.xml" --asic gfx1010 --pso "C:\Documents and Settings\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\Clone0/Pipeline0.gpso" --vert "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\vert.spv" --frag "C:\Users\AndreyOGL_D3D\Documents\RadeonGPUAnalyzer\Projects\Particles\frag.spv" 

Target GPU detected:

gfx1010 (RDNA)
	AMD Radeon RX 5700
	AMD Radeon RX 5700 XT
	AMD Radeon RX 5700 XT 50th Anniversary

Building for gfx1010... 
Error: failed to create graphics pipeline.
vkCreateGraphicsPipelines function returned error code: -3
Make sure that the provided pipeline state matches the given shaders.

To get more detailed information about this failure, try enabling Vulkan validation layers.
Warning: falling back to building using Vulkan offline mode (-s vk-spv-offline). The generated ISA disassembly and HW resource usage information might be inaccurate. To get the most accurate results, adjust the pipeline state to match the shaders and rebuild.

Falling back to vk-spv-offline mode...
Target GPU detected:

gfx1010 (RDNA)
	AMD Radeon RX 5700
	AMD Radeon RX 5700 XT
	AMD Radeon RX 5700 XT 50th Anniversary

Building for gfx1010... succeeded.
```

For 2.3 version I have no crash, Thanks you!.
